Extensions/timestamps.js

```diff
@@ -172,7 +183,7 @@ XKit.extensions.timestamps = new Object({
 
 			if (XKit.extensions.timestamps.in_search && !$("#search_posts").hasClass("posts_view_list")) {
 				var in_search_html = '<div class="xkit_timestamp_' + post_id + ' xtimestamp-in-search xtimestamp_loading">&nbsp;</div>';
-				post.find(".post_controls_top").prepend(in_search_html);
+				post.find(".post-info-tumblelogs").prepend(in_search_html);
```

Comment:
if you're adding things to the brick layout, you also need to call `XKit.interface.trigger_reflow()` (which is debounced, so should be fine to call naively in a loop)